src/Components/search/search.js

```diff
@@ -42,7 +58,7 @@ class Search extends React.Component {
                                 className="form-control searchbox__input"
                                 placeholder="Search movie..."/>
                         </form>
-                        {this.props.foundMovieList.length > 1 ? this.searchToolbar() : null }
+                        {this.props.foundMovieList.length > 1 && this.state.show ? this.searchToolbar() : null }
```

Comment:
Warunki lepiej wyniesc do nawiasu, dla czytelnosci :D ( (this.props.foundMovieList.length > 1) && this.state.show)